.github/workflows/sync-local-fallback-files.yml

```
@@ -0,0 +1,83 @@
+name: Sync runtime local fallback files
+on:
+  pull_request:
+  schedule:
+    # https://crontab.guru/every-night-at-midnight
+    - cron: "0 0 * * *"
+
+jobs:
+  sync-local-fallback-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          extensions: dom, iconv, json, libxml, zip
+          tools: composer
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Configure Composer cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Install Composer dependencies
+        run: composer install
+
+      - name: Configure git user
+        run: |
+          git config --global user.email "alain.schlesser@gmail.com"
+          git config --global user.name "Alain Schlesser"
+
+      - name: Create branch to base pull request on
+        run: |
+          git fetch
+          git branch -f sync-local-fallback-files origin/main
+          git checkout sync-local-fallback-files
+          git pull --no-rebase || true
+
+      - name: Sync local fallback files
+        run: composer sync-fallback-files
+
+      - name: Check if there are changes
+        id: changes
+        uses: UnicornGlobal/has-changes-action@v1.0.11
```

This action seems unnecessary as it's simply running `git status --porcelain` to determine if files have changed. This could be replaced with:

```suggestion
        run: echo ::set-output name=changed::$([[ -z $(git status --porcelain) ]] && echo "0" || echo "1")
```

---

Forgot to add:

This action adds a considerable amount of time to the overall execution time of the workflow. In [this check](https://github.com/ampproject/amp-toolbox-php/pull/58/checks?check_run_id=1844050880), you can see that it spends just about half of the time waiting for the Docker image to be built before it can continue.